en/complete_vehicles/atl_mantis_edu.md

```diff
@@ -31,26 +31,28 @@ The Mantis broadly features the following hardware components:
 The files to update the Mantis can currently be found on [GoogleDrive](https://drive.google.com/drive/u/1/folders/1LXW66IBGc_SgGLmk0ltl3R8IQYYVUvtz).
 
 Files include:
+- **autopilot-bootloader.px4**: Autopilot firmware to update bootloader (only to be used during first update).
+- **autopilot.px4**: Autopilot firmware
```

Comment:
Why did you re-order this? I had it in the same order as the headings below which is also the order how they need to be flashed! Otherwise it will fail.

Reply:
Ah, I did not consider that. I ordered them alphabetically as they will appear in a file browser.

Reply:
Should we make it an enumerated list then to make it clear, that there is an actual order?

Reply:
Yes, that sounds good.

Reply:
Done